src/main/java/view/GestureBridgeView.java

```diff
@@ -384,13 +398,41 @@ private void beginTextToSpeech() throws LineUnavailableException, IOException {
         String inputText = signLanguageTextArea.getText();
         String language = (String) languageBox.getSelectedItem();
         String languageCode = LanguageCodeMapper.getLanguageCode(language);
-        // This factory is generated to be a place-holder for the customization use case
-        AudioSettingsFactory audioSettingsFactory = new AudioSettingsFactory();
-        AudioSettings audioSettings = audioSettingsFactory.create(1.5, 2.0, false, 6.0); // Example settings
+        AudioSettings audioSettings = this.audioSettings;
         textToSpeechController.execute(inputText, languageCode, audioSettings);
 
     }
 
+    private VoiceSettingsView initializeSettingsView() {
+        return new VoiceSettingsView(this.audioSettings);
+    }
+
+    private void openSettings() {
+        VoiceSettingsView settingsView = initializeSettingsView();
+
+        CustomizeVoiceController customizeVoiceController = intializeVoiceCustomization(this,
+                settingsView);
+
+        settingsView.setCustomizeVoiceController(customizeVoiceController);
+
+    }
+
+    /**
+     * Initializes the Voice Customization module.
+     *
+     * @param gestureBridgeView the main application view for communication with presenters.
+     * @param settingsView the text to speech settings view for communication with presenters.
+     * @return the CustomVoiceController instance.
+     */
+    private static CustomizeVoiceController intializeVoiceCustomization(GestureBridgeView gestureBridgeView, VoiceSettingsView settingsView) {
+        AudioSettingsFactory audioSettingsFactory = new AudioSettingsFactory();
+        CustomizeVoiceDataAccessInterface dataAccessObject = new VoiceDataAccessObject();
+        CustomizeVoiceOutputBoundary outputBoundary = new CustomizeVoicePresenter(gestureBridgeView, settingsView);
+        CustomizeVoiceInputBoundary interactor = new CustomizeVoiceInteractor(dataAccessObject, outputBoundary,
+                                                                                audioSettingsFactory);
+        return new CustomizeVoiceController(interactor);
+    }
```

Comment:
The customize module and its controller are completely initialized within the view, not sure if that will cause dependency or code issues. Might need refactoring in the future